changelog/unreleased/enhancement-admin-settings-general-section

```diff
@@ -0,0 +1,6 @@
+Enhancement: Admin settings general section
+
+We've introduced the general section in the admin settings app, which displays information about the running
+front- and backend.
+
+https://github.com/owncloud/web/pull/8360
```

Comment:
Please also add the issue link https://github.com/owncloud/web/issues/8363 (I just synced it from jira to github)